### PR TITLE
Weekly review 31 January - 6 Feb 2015

### PR DESCRIPTION
**Pull request by anelda**
Missing information includes - where was the instructor training offered and what updates have been added to lessons

### CODE REVIEW

File: blog/2015/02/weekly-update-2015-02-06.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <tr>
+    <td>Other News:</td>
+    <td><a href="{{page.root}}/blog/2015/02/welcome-our-newest-instructors.html">Eight new instructors joined SWC</a></td>
+  </tr>
+</table>
+<!-- end excerpt -->
+<p><strong>Details</strong></p>
+<ul>
+  <li>
+    This week the first meeting of the new SWF steering committee took place
```
Comment (r-gaia-cs) [inline blog/2015/02/weekly-update-2015-02-06.html]: What about

~~~
This week the first meeting of the new SWF steering committee took place. Minutes will be publish soon.
~~~

File: blog/2015/02/weekly-update-2015-02-06.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  </li>
+  <li>
+    Juan Nunez-Iglesias, Harriet Dashnow, and Stefan van der Walt are looking for code nominations to be used in their book called Elegant SciPy - <a href=http://ilovesymposia.com/2015/02/04/call-for-code-nominations-for-elegant-scipy/">please get in touch with them </a>
```
Comment (r-gaia-cs) [inline blog/2015/02/weekly-update-2015-02-06.html]: There is a missing quote before the link. Need to replace with

~~~
    Juan Nunez-Iglesias, Harriet Dashnow, and Stefan van der Walt are looking for code nominations to be used in their book called Elegant SciPy - <a href="http://ilovesymposia.com/2015/02/04/call-for-code-nominations-for-elegant-scipy/">please get in touch with them</a>
~~~

File: blog/2015/02/weekly-update-2015-02-06.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  </li>
+ <li>
+    Sheldon McKay and Greg Wilson hosted a second post-workshop instructor debriefing attended by 10 instructors.    <td><a href="{{page.root}}/blog/2015/02/instructor-debriefing-2015-01-27.html">Read more about their lessons learnt</a>
+  </li>
+  <li>
+	Neil Davis summarised their experience and innovations introduced at the second SWC workshop held at <a href={{page.root}}/blog/2015/02/workshop-in-illinois.html">University of Illinois</a>. Their suggestions include: (i) get feedback at each break; (ii) introduce the idea that knowledge gained at SWC workshops can be applied tomorrow or on Monday as opposed to in future. He questioned whether SWC events advertisements are effective at creating a clear picture of what participants can expect.
```
Comment (r-gaia-cs) [inline blog/2015/02/weekly-update-2015-02-06.html]: There is a missing quote before the link. Need to replace with

~~~
	Neil Davis summarised their experience and innovations introduced at the second SWC workshop held at <a href="{{page.root}}/blog/2015/02/workshop-in-illinois.html">University of Illinois</a>. Their suggestions include: (i) get feedback at each break; (ii) introduce the idea that knowledge gained at SWC workshops can be applied tomorrow or on Monday as opposed to in future. He questioned whether SWC events advertisements are effective at creating a clear picture of what participants can expect.
~~~

**Comment by r-gaia-cs** [comment]
@anelda Thanks for the help with our weekly review. There are a few minor changes that need to be address.

**Comment by anelda** [comment]
Thanks @r-gaia-cs for reviewing and @gvwilson for editing

**Pull request merged**